region/norfair/west.json

```diff
@@ -1810,12 +1911,108 @@
               "id": 1,
               "strats": [
                 {
-                  "name": "Base",
+                  "name": "Crumble Shaft HiJump Climb",
+                  "notable": true,
+                  "requires": [
+                    "canNavigateHeatRooms",
+                    "HiJump",
+                    {"heatFrames": 500}
+                  ],
+                  "note": "Walljump up the right, then setup a walljump on the top-right crumble platform directly from the right wall."
+                },
+                {
+                  "name": "Crumble Shaft HiJumpless Climb",
+                  "notable": true,
+                  "requires": [
+                    "canNavigateHeatRooms",
+                    "canTrickyWalljump",
+                    {"heatFrames": 500}
+                  ],
+                  "note": "Walljump up the right, then walljump off the top-middle crumble platform then off the top-right one."
+                },
+                {
+                  "name": "Crumble Shaft Crumble Climb",
+                  "notable": true,
+                  "requires": [
+                    "canNavigateHeatRooms",
+                    "canCrumbleJump",
+                    {"heatFrames": 800}
+                  ],
+                  "note": "Do 9 successive crumble jumps up the platforms."
+                },
+                {
+                  "name": "Crumble Shaft Wall/Crumble Climb",
```

Comment:
obsoletes the 9 crumble jump strat

Reply:
True enough, but seeing how it's a notable strat, I think the other one should still stay in. Each of the 9 crumble jumps are individually easier than the single one here IMO.